viadot/sources/s3.py

```diff
@@ -106,21 +106,31 @@ def cp(self, from_path: str, to_path: str, recursive: bool = False):
         """
         self.fs.copy(path1=from_path, path2=to_path, recursive=recursive)
 
-    def rm(self, path: str):
+    def rm(self, paths: list[str]):
         """
         Deletes files in a path.
 
         Args:
-            path (str): Path to a file or folder to be removed. If the path refers to
-                a folder, it will be removed recursively.
+            path (list[str]): Path to a file or folder to be removed. If the path refers to
+                a folder, it will be removed recursively. Also the possibilty to delete multiple files at once by
+                passing the individual paths as a list of strings.
+        ```python
+        from viadot.sources import S3
+        s3 = S3()
+        s3.rm(
+            paths = ['path_first_file', 'path_second_file']
```

Comment:
don't put spaces between keywords

Reply:
Updated in:

- https://github.com/dyvenia/viadot/pull/603/commits/e0316bb746f695f87a7cc71a731146eeacb1e1f7